tools/run_tests/run_tests.py

```diff
@@ -166,6 +174,37 @@ def supports_multi_config(self):
   def __str__(self):
     return self.make_target
 
+def gyp_test_paths(travis, config=None):
+  binaries = get_c_tests(travis, 'c')
+  out = []
+  for target in binaries:
+    if config is not None:
```

Comment:
Always condense

if (A):
  if (B):
    (statements)

into

if (A) and (B):
  (statements)

.